src/mscorlib/src/System/Text/DecoderBestFitFallback.cs

```diff
@@ -34,13 +31,7 @@ public override DecoderFallbackBuffer CreateFallbackBuffer()
         }
 
         // Maximum number of characters that this instance of this fallback could return
-        public override int MaxCharCount
-        {
-            get
-            {
-                return 1;
-            }
-        }
+        public override int MaxCharCount => 1;
```

Comment:
We want to get the sources of types like this unified between CoreCLR and CoreRT (https://github.com/dotnet/coreclr/issues/972). There is one set of cleanups applied here and different set of cleanups applied in CoreRT. I would be useful to get the code unified first, and apply the cleanups only after that (or do the cleanups in CoreRT and then copy the cleaned up version over to CoreCLR).